app/templates/components/file-upload.html

```diff
@@ -25,7 +26,8 @@
       {% endif %}
     </label>
     {{ field(**{
-      'class': 'file-upload-field'
+      'class': 'file-upload-field',
+      'accept': allowed_file_extensions
```

Comment:
I’d prefer `allowed_file_extensions` was a list, and we transform it into the attribute string once here, for example
```suggestion
      'accept': allowed_file_extensions|prepend('.')|join(',')
```
(You’d need to write the `prepend` Jinja filter yourself since there isn’t a built in one)

Reply:
What about using [formatted_list](https://github.com/alphagov/notifications-utils/blob/b1b9c1aa1da6ab5eef39d5b35089341f650748b4/notifications_utils/formatters.py#L195) for that instead of writing a new filter?

Reply:
Nice thinking 🧠

Reply:
So it turns out we couldn't use it because it assumes a conjunction so even if you set that to `''`, it has the spaces around it. Instead I (kind of) took your advice and added a `format_list_items` filter which just lets you apply a [format string](https://docs.python.org/3/library/string.html#format-string-syntax) and whatever arguments you want to each item in an iterator.